nucypher/characters/lawful.py

```diff
@@ -967,12 +967,19 @@ def run(
         #
 
         if emitter:
-            emitter.message("Starting services", color="yellow")
+            emitter.message("Starting services...", color="yellow")
 
         if discovery and not self.lonely:
             self.start_learning_loop(now=eager)
             if emitter:
-                emitter.message(f"✓ Node Discovery ({self.domain})", color="green")
+                emitter.message(f"✓ P2P Networking ({self.domain})", color="green")
+
+        if transaction_tracking:
+            # Uncomment to enable tracking for both chains.
+            # mainnet.tracker.start(now=False)
+            polygon.tx_machine.start(now=False)
```

Comment:
The logical grouping of initializing all the async tasks for the lifecycle of the program is good, and I'd hesitate before making an exception for this one.

Reply:
Not sure I follow...

Reply:
> Does the tx_machine need to be stopped as part of the Ursula.stop(...) method?

Not sure about this one but I think it's low impact since the runtime lifecycle normally does not require `stop` to be invoked.  In tests we have control over individual async tasks which is sufficient.  that being said we can include programmatic stopping functionality for completeness

> `.start`

There benefit to keeping calls to `.start` in this function is that we're explicitly keeping all async task initiation for production runtimes inside one scope.  Since `BlockchainInterfaceFactory` operates with an internal cache my initial hesitation is that by moving `.start` to a lower layer we'd be implicitly firing async tasks in tests that we may loose control over.

Reply:
Pushes cc22ed134bb5eeef1091c73cfa087c78615b0e41 which allows unexposes eager atxm startup functionality from ursula ans allows atxm to manage it's own async task internally.

Also Included commit d75e20f87c62349a8b04030199a30c1be742afd2 which explicitly returns `None` from all `SimpleTask` subclass start and stop calls.